torch/jit/_monkeytype_config.py

```diff
@@ -0,0 +1,115 @@
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+# mypy: ignore-errors
```

Comment:
What need to be suppressed? Normally we should avoid global ignore.

Reply:
Now removed